engine/security/apparmor.md

```diff
@@ -46,7 +50,11 @@ profile docker-default flags=(attach_disconnected,mediate_deleted) {
   file,
   umount,
 
-  deny @{PROC}/{*,**^[0-9*],sys/kernel/shm*} wkx,
+  deny @{PROC}/* w,   # deny write for all files directly in /proc (not in a subdir)
```

Comment:
I don't love the idea of having the profile itself duplicated in the docs. Maybe a link to the profile's source file on Github?